app/react/Uploads/components/ImportPanel.js

```diff
@@ -103,11 +126,26 @@ export class ImportPanel extends Component {
 
   renderContents() {
     const { uploadProgress, importStart, importProgress } = this.props;
-    if (uploadProgress) {
+    if (uploadProgress && !this.state.showError) {
       return this.renderUploadProgress();
     }
 
-    if (importStart || importProgress) {
+    if (this.state.showError) {
+      return (
+        <>
+          <div className="alert alert-danger">
+            <Icon icon="exclamation-triangle" />
+            <div className="force-ltr">
+              <Translate>The import process threw an error:</Translate>
+            </div>
+            <Icon style={{ cursor: 'pointer' }} icon="times" onClick={this.closeError} />
+          </div>
+          <StackTrace message={this.importError.error} validations={this.importError.validations} />
+        </>
+      );
+    }
```

Comment:
This section is the same in `ImportProgress`, I think the solution is not to duplicate this code but diagnose why it was not working correctly

Reply:
I have update the codebase. It seems the server returns `importError.prettyMessage` instead of `importError.message`